video/out/wldmabuf/ra_wldmabuf.c

```diff
@@ -19,21 +19,44 @@
 #include "video/out/gpu/ra.h"
 #include "ra_wldmabuf.h"
 
+struct priv {
+    struct vo* vo;
+};
+
 static void destroy(struct ra *ra)
 {
+    talloc_free(ra->priv);
+}
+
+bool ra_compatible_format(struct ra* ra, uint32_t drm_format, uint64_t modifier)
+{
+    struct priv* p = ra->priv;
+    struct vo_wayland_state *wl = p->vo->wl;
+    const wl_format *formats = wl->format_map;
+
+    for (int i = 0; i < wl->format_size / 16; ++i)
```

Comment:
Where does the magic 16 come from? Make it a named constant.

Reply:
It's the size of the struct that stores format info. fixed.